### DISCUSSION TITLE
Customize navigator keys

### DISCUSSION THREAD

**Discussion by cloin** [ansible/ansible-navigator#588] [Ideas]
It would be really nice to have some kind of key mappings file to allow to change the default keymappings for things like scrolling and moving back to the previous page in navigator, etc. 

I'm fighting with capturing the <kbd>Esc</kbd> key in navigator when being used in the integrated terminal of vs code in a web browser.

**Comment by ssbarnea** [comment]
We already talked about this and we aim to make them configurable. The issue is that it will take time to implement it.

As you noted, the use inside web-browser does impose some challenges but that is also a use-case we want to support. If you manage to find a workaround we would clearly welcome a pull-request.
